src/effects/DynamicRangeProcessorHistoryPanel.cpp

```diff
@@ -246,7 +246,8 @@ void DynamicRangeProcessorHistoryPanel::InitializeForPlayback(
    const size_t maxQueueSize = DynamicRangeProcessorHistory::maxTimeSeconds *
                                sampleRate / leastPacketSize;
    mPacketBuffer.reserve(maxQueueSize);
-   mOutputQueue.reset(new DynamicRangeProcessorOutputPacketQueue(maxQueueSize));
+   mOutputQueue =
+      std::make_shared<DynamicRangeProcessorOutputPacketQueue>(maxQueueSize);
```

Comment:
Mh, this line fails on MacOS x86_64, with the error
```
/Applications/Xcode_15.0.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk/usr/include/c++/v1/__memory/shared_ptr.h:269:8: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.13 or newer
```
So it's not happy about the use of `std::align_val_t`. Probably due to
```cpp
template <typename T> class LockFreeQueue
{
public:
 ...
private:
   // Align the two atomics to avoid false sharing
   // mStart is written only by the reader, mEnd by the writer
   NonInterfering<std::atomic<size_t>> mStart { 0 }, mEnd { 0 };
};
```
which inherits from `NonInterferingBase`,
```cpp
NonInterferingBase {
   static void *operator new(std::size_t count, std::align_val_t al);
   static void operator delete(void *ptr, std::align_val_t al);
   ...
};
```
But `MeterPanelQueue` also uses this class, and this never caused that build problem 🤷 
```cpp
using MeterUpdateQueue = LockFreeQueue<MeterUpdateMsg>;
```
My only yet feeble guess is that the memory layout of `MeterUpdateMsg` differs from that of `DynamicRangeProcessorOutputPacket` in a fortunate way.
Any idea @vsverchinsky ?